File: BB.Web/app/Model/Products/ProductDetails.ts
import r = require("./Rating");
import Rating = r.Rating;
import pc = require("./ProductCharacteristic");
import ProductCharacteristic = pc.ProductCharacteristic;

export class ProductDetails {
    public Description: string;
    public Characteristics: ProductCharacteristic[];
    public Comments: Comment[];
    public Ratings: Rating[];
    public Images: string[];
}

File: BB.Core/Model/Products/ProductDetails.cs
using System.Collections.Generic;

namespace BB.Core.Model.Products
{
    public class ProductDetails
    {
        public string Description { get; set; }

        public List<ProductCharacteristic> Characteristics { get; set; }

        public List<Comment> Comments { get; set; }

        public List<Rating> Ratings { get; set; }

        public List<string> ImageURLs { get; set; }
    }
}
